insights/util/symbolic_name.py

```diff
@@ -0,0 +1,29 @@
+"""
+A method to get the spec name of some special symbolic_name in the uploader.json
+"""
+
+
+def special_spec_name_of_symbolic_name(sname):
+    # match a spec name to a symbolic name
+    # some symbolic names need to be renamed to fit specs
+
+    spec_conversion = {
```

Comment:
HI @xiangce , do we consider to move this dict out of this function? It can avoid the redefine of this dict in each function call.